src/views/network/Network.vue

```diff
@@ -1,92 +1,109 @@
-                                        {{ requests[uid].headers.url.pathname }}
-                                    </v-list-item-title>
-                                    <v-list-item-subtitle class="origin pb-2">{{ requests[uid].headers.url.origin }}</v-list-item-subtitle>
-                                </v-list-item-content>
-                                <div class="mr-3">
-                                    <table>
-                                        <tr class="request-info">
-                                            <td class="request-status" :class="statusColor(requests[uid].status)">
-                                                <span v-if="requests[uid].status" class="font-weight-bold">{{ requests[uid].status }}</span>
-                                                <v-icon v-else class="request-loading">mdi-timelapse</v-icon>
-                                            </td>
-                                            <td class="request-timestamp font-weight-bold text-right">{{ formatTimestamp(requests[uid].timestamp) }}</td>
-                                        </tr>
-                                        <tr v-if="requests[uid].response" class="v-list-item__subtitle request-info">
-                                            <td>{{ formatDuration(requests[uid].response.timestamp - requests[uid].timestamp) }}</td>
-                                            <td class="text-right">{{ filesize(requests[uid].response.body.size) }}</td>
-                                        </tr>
-                                    </table>
+    <pane>
+        <v-layout wrap v-if="search_enabled">
+            <v-flex ml-3 mb-4 xs12 sm6 md3>
+                <v-text-field v-model="search_filter"
+                              hide-details
+                              prepend-inner-icon="mdi-magnify"
+                              single-line
+                              ref="search"
+                              clearable
+                              @click:clear="clear_search"
+                              placeholder="Search ..."></v-text-field>
+                <v-spacer></v-spacer>
+            </v-flex>
+        </v-layout>
+        <splitpanes class="network-panel default-theme fill-height">
+            <pane style="min-width: 250px" :size="100 - detail_size">
+                <!--suppress HtmlUnknownAttribute -->
+                <div ref="scroll" class="network-container absolute-expand overflow-y-auto">
+                    <v-list v-if="session_list.length" dense>
+                        <v-list-item-group v-model="selected" color="primary">
+                            <div v-for="(s,i) in session_list" :key="i">
+                                <div v-if="search_filter === null || search_filter.trim().length <= 0" class="text-center text--text font-weight-bold pt-4">{{ formatTimestamp(s.timestamp, true) }}</div>
+                                <div v-for="uid in search_requests(s)" :key="uid">
```

Comment:
You don't have to filter the sessions if you're always hiding the session title.
Just filter the requests, like this:
```vue
<div v-for="uid in s.requests" :key="uid">
    <v-list-item v-if="search_requests(requests[uid])" class="request-item" two-line>
```